Ch06/CH06.md

```diff
@@ -735,46 +734,52 @@ for us to tell which version of the document is newer than the other. This is a
-you want it to resolve conflicts as they happen. By default, we resolve conflicts in favor of the most 
+The question is this &mdash; how are we going to handle such a case? Let's generate a conflict manually and see how this is handled. 
+But first, we need to disable automatic conflict resolution so we can observe what is going on behind
+the scenes. In the Studio, go to `Settings`, `Conflict Resolution`. This is where you let RavenDB know how
+you want it to resolve conflicts when they happen. By default, we resolve conflicts in favor of the most 
 recent version^[Note that different nodes may have different clocks. RavenDB doesn't attempt to compensate
 for this and will use the latest timestamp from the conflicts in its automatic resolution.]. You can see
 what this look like in Figure 6.14.
 
 ![Configuring server side conflict resolution policy in the Studio](./Ch06/img14.png)
 
 Uncheck the option to automatically resolve to the latest version and click on `Save`. Now we are ready
-to create some conflicts. As you'll shortly see, conflicts occur when there is effectively a race in the 
+to create some conflicts. As you'll see shortly, conflicts occur when there is effectively a race in the 
 cluster between modifications to the same document on different nodes. This is a bit hard to reproduce,
 so we'll take the heavy-handed approach and reproduce a conflicts by shutting down nodes and modifying
 documents independently on each node. Here's how we'll do it:
 
-* Kill node `B`. (It's running on `127.0.0.2`.) You can do that by simply closing the console window that's running it.
-* Modify the `users/1` document on node `A` so its name property will be "John," and save the document. You can see how this will look in 
-  Figure 6.15.
-* Kill node `A`. (It's running on `127.0.0.1`.) 
-* Bring up node `B` by executing `./Server/Raven.Server.exe --ServerUrl=http://127.0.0.2:8080 --Logs.Path=Logs/B --DataDir=Data/B`.
-* Modify the `users/1` document on node `B` so its name property will be "Doe," and save the document. You can see how this will look in
-  Figure 6.16.
-* Bring up node `A` by executing `./Server/Raven.Server.exe --ServerUrl=http://127.0.0.1:8080 --Logs.Path=Logs/A --DataDir=Data/A`.
+* Kill node `B`. (It's running on `127.0.0.2`) You can do that by simply closing the console window that's running it.  
+* Modify the `users/1` document on node `A` so its name property will be "John," and save the document. You can see how this will look in Figure 6.15.  
+* Kill node `A`. (It's running on `127.0.0.1`)  
+* Bring up node `B` by executing:  
+    ./Server/Raven.Server.exe`  
+    --ServerUrl=http://127.0.0.2:8080`  
+    --Logs.Path=Logs/B`  
+    --DataDir=Data/B  
+* Modify the `users/1` document on node `B` so its name property will be "Doe", and save the document. You can see how this will look in Figure 6.16.  
+* Bring up node `A` by executing:  
+    ./Server/Raven.Server.exe`  
```

Comment:
Add space before `

Reply:
Also need to check here how this looks like after building the book

Reply:
done